public/javascripts/SVValidate/src/Main.js

```diff
@@ -209,6 +209,486 @@ function Main (param) {
             svv.keyboard.disableKeyboard();
             $(".tool-ui").css('opacity', 0.5);
         });
+
+        // Validate mission tutorial descriptor.
+        // See MissionTutorial.js for documentation.
+        const tutorialDescriptor = {
```

Comment:
I'm not exactly sure where this should live, but probably not in `Main.js`!

Reply:
I can make a separate file for it and bundle it along. Does that work?

Reply:
Either in a separate file or in the `MissionTutorial.js` file would be fine iwth me!

Reply:
Moved to MissionStartTutorial.js